contents/blog/posthog-vs-amplitude.mdx

```diff
@@ -16,57 +16,50 @@ import { ComparisonTable } from 'components/ComparisonTable'
 import { ComparisonHeader } from 'components/ComparisonTable/header'
 import { ComparisonRow } from 'components/ComparisonTable/row'
 
-Want to know how PostHog and Amplitude compare? If you remember nothing else, remember these two things:
+Amplitude and PostHog are superficially similar. 
 
-1. **Amplitude** is a product analytics tool built for non-technical users and data analysts.
+They both offer product analytics and a range of additional tools, like feature flags, A/B testing, and session replay. 
 
-2. **PostHog** is an all-in-one platform built for engineers that replaces Amplitude _and_ numerous other tools.
+But, deep down, Amplitude is an analytics tool with some extra bits grafted on. In contrast, PostHog is built from the ground up as an all-in-one platform for building successful products. Here's how they compare.
```

Comment:
INtro is too long IMO. Can we be more to the point?

Reply:
I could go back to just not having an intro again, tbh. It doesn't feel essential to have it.